gcp-observability/build.gradle

```diff
@@ -48,7 +48,8 @@ dependencies {
 
     runtimeOnly libraries.opencensus.impl
 
-    testImplementation project(':grpc-testing'),
+    testImplementation project(':grpc-context').sourceSets.test.output,
```

Comment:
I wonder why this was needed: if transitive dependency wouldn't have included `grpc-context` with these attributes.

Reply:
This was needed for `StaticTestingClassLoader` which is defined in `grpc-context`. 
Transitive dependency through `grpc-testing` includes `grpc-context` as `runtimeOnly` dependency, so adding this an explicit dependency resolved `symbol not found` error.